corehq/apps/app_manager/templates/app_manager/partials/menu/appnav_menu.html

```diff
@@ -67,3 +67,35 @@
         </li>
     {% endif %}
 </ul>
+
+<div id="reorder_modules_modal" class="modal fade" tabindex="-1" role="dialog">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+                <h4 class="modal-title">
+                    {% trans "Your modules were reordered" %}
+                </h4>
+            </div>
+            <form action="{% url "reorder_child_modules" domain app.id %}"
+                  method="post">
+                {% csrf_token %}
+                <div class="modal-body">
+                    <p class="lead">
+                        {% trans "Your modules were reordered" %}
+                    </p>
+                    <p>
+                        Click below to move these child modules under their parents.
+                    </p>
+                </div>
+                <div class="modal-footer">
+                    <button type="submit" class="disable-on-submit btn btn-danger">
```

Comment:
Is this a dangerous action or is this what we want them to do?

Either way I'd add an explicit cancel button so people don't feel trapped.

Reply:
I am trying to force them to make that change, though I was also thinking about a "cancel", since popping this up every pageload and not allowing them to reorder modules is already a pretty aggressive way of forcing it.

Reply:
I'd mark it as `btn-primary` then.